samples/sampler/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 The following is a curated list of changes in the Enact Sampler, newest changes on the top.
 
+## unreleased
+
+- Updated `babel-loader` version to `^9.1.2`.
+- Updated the minimum version of Node to `14.15.0`.
```

Comment:
Could you explain this? I cannot find this change in this PR.

Reply:
Sorry, it was a change I undid and I forgot to delete this. Thank you!